raphtory/src/core/entities/properties/graph_meta.rs

```diff
@@ -60,7 +60,9 @@ impl GraphMeta {
         if is_static {
             Ok(self.constant_mapper.get_or_create_id(name))
         } else {
-            self.temporal_mapper.get_or_create_and_validate(name, dtype)
+            self.temporal_mapper
+                .get_or_create_and_validate(name, dtype)
+                .map_err(|e| e.into())
```

Comment:
Maybe we should just move the GraphError enum to `raphtory-api` instead of having to worry about different error types?

Reply:
I don't want to do this as part of this PR